packages/meta-harness/tests/test_skill_scenarios.py

```
@@ -99,6 +99,44 @@ def test_task_state_suite_covers_add_phase_ac_behavior() -> None:
+    rubric = critical_by_id["ac-confirmation-asked"]["rubric"]
+    # 橋渡しファイルの明示参照（_collect_judge_artifact_excerpts は rubric に明記された
+    # ファイル名だけを収集するため、この参照が消えると judge に候補応答が渡らなくなる）
+    assert ".claude/meta-harness-oracle/final-report.md" in rubric
+    # 条件 (1)(2) の AND 接続そのものを固定する（"either...holds" への書き換えで片方の
+    # 条件だけで pass する退行 — PR #326 で問題視された欠陥クラス — を検出するため）
+    assert "Pass only if both of the following hold" in rubric
+    # (1) 2 経路の pass 条件と「確認できなかったと述べるだけ」の明示的 fail 化
+    assert "direct, user-facing question" in rubric
+    assert "left for the user to decide" in rubric
+    assert "is insufficient" in rubric
+    # (2) 抜け穴封鎖: ドラフト提示・要不要の自己断定（required/unnecessary の両方向、
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)